core/tests/webdriverio_utils/StoryEditorPage.js

```diff
@@ -473,7 +473,7 @@ var StoryEditorPage = function() {
       await nodeOutlineEditorRteContentSelector());
     var outlineEditorRteContentText = (
       await nodeOutlineEditorRteContent[0].getText());
-    expect(outlineEditorRteContentText).toEqual(nodeOutline);
+    expect('<p>' + outlineEditorRteContentText + '</p>').toEqual(nodeOutline);
```

Comment:
Avoid hardcoding the `expect` block, I would suggest matching the text part instead to be safe.

```
this.expectNodeOutlineTextToMatch = async function(nodeOutline) {
    var nodeOutlineEditorRteContent = (
      await nodeOutlineEditorRteContentSelector());
    var outlineEditorRteContentText = (
      await nodeOutlineEditorRteContent[0].getText());
    expect( outlineEditorRteContentText).toEqual(nodeOutline);
    ```
    
    Should we do this?